controllers/redis_controller.go

```diff
@@ -650,11 +651,12 @@ const (
 	baseName               = "redis"
 	redisVersion           = "6.0.8-alpine"
 	imageName              = baseName + ":" + redisVersion
-
-	version        = "v1"
-	kubeconfigPath = "/Users/mkfd/.kube/config"
+	version                = "v1"
 )
 
+var home, _ = os.UserHomeDir()
```

Comment:
What about something like this?

```suggestion
func findKubeConfig() (string, error) {
	env := os.Getenv("KUBECONFIG")
	if env != "" {
		return env, nil
	}
	path, err := homedir.Expand("~/.kube/config")
	if err != nil {
		return "", err
	}
	return path, nil
}
```

Reply:
This is probably the better way to do it. Then the user has the option to either pass the config file as an env-variable or use the "default" path.

Reply:
Since the kubeConfigPath variable is declared outside of a function, where would it then make sense to check for the return of the findKubeConfig() function?